pantheon-bundle/frontend/src/app/__snapshots__/searchBeta.test.tsx.snap

```diff
@@ -205,7 +210,7 @@ exports[`SearchBeta tests should render default Search component 1`] = `
               width=""
             >
               <SelectOption
-                className=""
+                className="filters-bar__option filters-bar__option-content-type filters-bar__option--concept"
```

Comment:
I'd replkace `filters-bar` with `dropdown-filter` in each of these

`option` is part of the `dropdown-filter` component, unless I'm mistaken?

Reply:
And same with the below.